packages/gatsby-recipes/package.json

```diff
@@ -62,14 +46,9 @@
     "mitt": "^1.2.0",
     "mkdirp": "^0.5.1",
     "node-fetch": "^2.5.0",
-    "normalize.css": "^8.0.1",
     "pkg-dir": "^4.2.0",
     "prettier": "^2.0.5",
     "prop-types": "^15.6.1",
-    "property-information": "5.5.0",
-    "react-circular-progressbar": "^2.0.0",
-    "react-icons": "^3.0.1",
-    "react-reconciler": "^0.25.1",
     "remark-mdx": "^2.0.0-next.4",
     "remark-mdxjs": "^2.0.0-next.4",
     "remark-parse": "^6.0.3",
```

Comment:
Those are additional candidates for bundling - I wasn't sure if those cause problems or not, so for now those are not bundled